include/aspect/utilities.h

```diff
@@ -332,12 +332,12 @@ namespace aspect
         /**
          * Interpolation functions to access the data.
          */
-        std::vector<Functions::InterpolatedUniformGridData<dim> *> data;
+        std::vector<Functions::InterpolatedTensorProductGridData<dim> *> data;
```

Comment:
Can you instead just make it a `std::vector<Function<dim>*>`, for reasons that will become clear below?